test/global.test.ts

```diff
@@ -93,9 +94,30 @@ export async function createBuildTask(folder: vscode.WorkspaceFolder): Promise<v
 
         allTasks.push(buildTask);
         await workspaceTasks.update('tasks', allTasks, vscode.ConfigurationTarget.WorkspaceFolder);
+        console.log(`${testName}: Build task created`);
+        verifyBuildTask(testName, folder);
     }
 }
 
+function verifyBuildTask(testName: string, folder: vscode.WorkspaceFolder): void {
+    const maxTry = 5;
+    let i = 0;
+    let buildTaskPresent = isBuildTaskPresent(folder);
+
+    while (!buildTaskPresent && i < maxTry) {
```

Comment:
Once we figure out and fix the real root cause let's remove this.

Reply:
Yes. that is the plan.